hack/testing/test-viaq-data-model.py

```diff
@@ -142,7 +142,7 @@ def empty(thing):
         if xx not in dd['_source']:
             print "Error: input does not have the field [%s]" % xx
             sys.exit(1)
-        if not match[xx] == dd['_source'][xx]:
+        if not match[xx] == dd['_source'][xx] or not match[xx].decode('utf-8') == dd['_source'][xx.decode('utf-8')]:
```

Comment:
I doubt that this is the problem . . .

Reply:
Ok.  According to this output [1], certainly `stream` is an issue.  Do you think once stream is dropped, the comparison returns SUCCESS?

[1] - http://pastebin.test.redhat.com/781238

Reply:
> Do you think once stream is dropped, the comparison returns SUCCESS?

Yes

Reply:
All right.  Let me undo the change...